move/axelar_gateway/sources/types/gateway_data.move

```diff
@@ -86,6 +111,14 @@ public (package) fun signers_mut(self: &mut GatewayDataV0): &mut AxelarSigners {
     &mut self.signers
 }
 
+public (package) fun version_control(self: &GatewayDataV0): &VersionControl {
+    &self.version_control
+}
+
+public (package) fun version_control_mut(self: &mut GatewayDataV0): &mut VersionControl {
```

Comment:
is this even needed?

Reply:
yes

Reply:
well, don't leave me hanging, where is it used? :p

Reply:
oh it is not used currently, but it will be if we want to update version control upon a package upgrade, but we could remove it for now.